src/components/Modal/Modal.tsx

```diff
@@ -10,6 +10,7 @@ const ModalOverlay = styled.div<{ noBackground?: boolean }>`
   padding: 15px;
   background: rgba(0, 0, 0, 0.4);
   border-radius: 12px;
+  z-index: ${({ zIndex }) => zIndex && '1'};
```

Comment:
```suggestion
  ${({ zIndex }) => zIndex && 'z-index: 1;'}
```

Reply:
I would also let `zIndex` to be number or if you want to leave as boolean then change prop to something that would match scenario like `isComponentOnTop` or else rather than value name, reason because you can see how other params that are matching css are accepting same values as css